QUEST.tsv

```diff
@@ -1948,156 +1948,156 @@ QUEST_20150428_001947	$One more thing. Do not consume too much at once, it could
+QUEST_20150511_002000	$We never know when it will release spores. If we don't stop it, the whole camp could be annihilated. {nl}I have no way to keep safe from it. I can't stop coughing…
+QUEST_20150511_002001	$The training officer completed the training at the camp in the woods of the attached bridge, in accordance with the order to prepare for future disasters. {nl}I hereby grant this certificate. 
+QUEST_20150511_002002	$The Mausoleum of Zachariel has never allowed anyone entry. {nl}If the object was not for the world to know of, then we could've hidden it so that none could find it, but instead we just built a giant tomb to protect it.{nl}Is this the great king's conceit or does it hold a different meaning?
+QUEST_20150511_002003	$For the first step: recharge the magical essence I gave to you. Look for where magic is condensed at Pavydo Mixed Forest. {nl}You won't be able to use the magical essence right away, however, since it has also absorbed some malign energy. {nl}For that, you'll need to collect Operor saliva and purify it at the altar of purification at the Wasteland of Renistu.
+QUEST_20150511_002004	$Name: High Raven{nl}Sex: Male{nl}Occupation: Merchant{nl}List of items: luxurious goods, weapons {nl}{nl}In accordance with kingdom laws, you may enter.
+QUEST_20150511_002005	$If you're seeing this you either didn't pay me in full or did something I told you not to do. {nl}I have cursed this letter. {nl}I've no idea what's gonna happen to you from now on, but know that my curse is absolute.
+QUEST_20150511_002006	$The one who fell while investigating the knowledge. {nl}Return to the Goddess by becoming a part of the canyon. {nl}The saddened mind breaks the gold stones of the canyon and the poured alcohol soaks into the ground on behalf of the tears.
+QUEST_20150511_002007	$Eventually, Ruklys had failed. {nl}He was destroyed and broken by those who were jealous of him. {nl}He left the Goddess in my hands. {nl}
+QUEST_20150511_002008	$Every object possess the inner strength of its possessor. {nl}And to add to the magical interpretations on that, they could also transform the shapes of their possessors. {nl}Objects belonging to demons could do it with relative ease. (The same applies to the sacred areas, but don't even dare doing that in those places.)
+QUEST_20150511_002009	$But that's still just a theory despite the efforts of many frontiers. {nl}The possessor's inner strength and interpretations can never be fully understood if it were not for the possessor himself. {nl}That's why we hope for the blessing from the Goddess or the wisdom from the possessor for the knowledge we lack.
+QUEST_20150511_002010	$I shall name you Revelator, my representative sent out to seek help from others. {nl}We can't be deceived any more. {nl}We have to protest against the violence behind the mask that claims to protect us.
+QUEST_20150511_002011	$We believe your help is the main factor to our success so that our efforts were not made in vain.
```

Comment:
The tense seems wrong here. 
"so that our efforts were not made in vain" -> "so that our efforts will not end up being in vain"